Source/JavaScriptCore/b3/air/AirAllocateRegistersAndStackAndGenerateCode.cpp

```diff
@@ -248,34 +258,34 @@ ALWAYS_INLINE bool GenerateAndAllocateRegisters::assignTmp(Tmp& tmp, Bank bank,
 
     auto markRegisterAsUsed = [&] (Reg reg) {
         if (Arg::isAnyDef(role))
-            m_clobberedToClear.clear(reg);
+            m_clobberedToClear.ignore(reg);
         // At this point, it doesn't matter if we add it to the m_namedUsedRegs or m_namedDefdRegs. 
         // We just need to mark that we can't use it again for another tmp.
-        m_namedUsedRegs.set(reg);
+        m_namedUsedRegs.preserve(reg);
     };
 
     bool mightInterfere = m_earlyClobber.numberOfSetRegisters() || m_lateClobber.numberOfSetRegisters();
 
     auto interferesWithClobber = [&] (Reg reg) {
         if (!mightInterfere)
             return false;
-        if (Arg::isAnyUse(role) && m_earlyClobber.get(reg))
+        if (Arg::isAnyUse(role) && m_earlyClobber.preserves(reg))
             return true;
-        if (Arg::isAnyDef(role) && m_lateClobber.get(reg))
+        if (Arg::isAnyDef(role) && m_lateClobber.preserves(reg))
```

Comment:
This API name change especially makes no sense here. This is a register set that represents what a patchpoint clobbers. Why are we asking it if it "preserves" anything?